DSharpPlus/Entities/Interaction/Application/DiscordApplicationCommandOptionChoice.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Newtonsoft.Json;
+
+namespace DSharpPlus.Entities
+{
+    /// <summary>
+    /// Represents a command parameter choice for a <see cref="DiscordApplicationCommandOption"/>.
+    /// </summary>
+    public sealed class DiscordApplicationCommandOptionChoice
+    {
+        /// <summary>
+        /// Gets the name of this choice parameter. 
+        /// </summary>
+        [JsonProperty("name")]
+        public string Name { get; set; }
+
+        /// <summary>
+        /// Gets the value of this choice parameter. 
+        /// </summary>
+        [JsonProperty("value")]  //TODO: JsonConverter may be necessary for this depending on the string.
```

Comment:
A converter shouldn't be necessary here. From my testing, the values are always sent as either a `long` or a `string`.

Reply:
It says [on the docs](https://discord.com/developers/docs/interactions/slash-commands#applicationcommandoptionchoice) that it may be either a string or int. I'm not sure if the string could be parsed as a snowflake.

Reply:
There should be any need to parse as a snowflake, this just represents a choice for a slash command option, so the values are provided by the user and there shouldn't be anything provided by discord in there.